routes/socket/game/election.js

```diff
@@ -260,6 +260,27 @@ const enactPolicy = (game, team, socket) => {
 								let list = seatedPlayers.filter((player, i) => i !== presidentIndex && !seatedPlayers[i].isDead);
 
 								game.gameState.timedModeEnabled = false;
+								
+								game.chats.push({
+									gameChat: true,
+									timestamp: new Date(),
+									chat: [
+										{
+											text: 'The timer has forced '
+										},
+										{
+											text: game.general.blindMode
+												? `${replacementNames[presidentIndex]} {${presidentIndex + 1}} `
+												: `${seatedPlayers[presidentIndex].userName} {${presidentIndex + 1}}`,
+											type: 'player'
+										},
+										{
+											text: powerToEnact[1]==='The president must examine the top 3 policies.'

```

Comment:
Lint please (there should be spaces around "===")

Reply:
there we go